src/patternfly/components/Sequence/sequence.scss

```diff
@@ -0,0 +1,350 @@
+
+  // Content Status Color
+  --pf-c-sequence__content-status--Color: var(--pf-global--Color--200);
+
+  // Content Status Font
+  --pf-c-sequence__content-status--FontSize: var(--pf-global--FontSize--xs);
+  --pf-c-sequence__content-status--FontWeight: var(--pf-global--FontWeight--regular);
+
+  position: relative;
+  display: flex;
+  justify-content: center;
+  height: var(--pf-c-sequence--Height);
+  overflow-y: hidden;
+
+  // remove default border styling
+  button {
+    padding: 0;
+    background-color: transparent;
+    border-top: 0;
+    border-bottom: 0;
+
+    &:focus {
+      outline: none;
+    }
+  }
+
+  button::-moz-focus-inner {
+    border: 0;
+  }
+
+  // remove scrollbar works on Chrome and Safari
+  ::-webkit-scrollbar {
+    display: none;
+    width: 0;
+  }
+}
+
+.pf-c-sequence__control {
+  top: 0;
+  bottom: 0;
+  padding-top: var(--pf-c-sequence__control--PaddingTop);
+  padding-bottom: var(--pf-c-sequence__control--PaddingBottom);
+
+  &::before {
+    content: "";
+    border-top: var(--pf-c-sequence__control--BorderTopWidth) solid var(--pf-c-sequence__control--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--BorderBottomWidth) solid var(--pf-c-sequence__control--BorderBottomColor);
+  }
+
+  &.pf-m-hover,
+  &:hover {
+    border-top: var(--pf-c-sequence__control--hover--BorderTopWidth) solid var(--pf-c-sequence__control--hover--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--hover--BorderBottomWidth) solid var(--pf-c-sequence__control--hover--BorderBottomColor);
+  }
+
+  &.pf-m-active,
+  &:active {
+    border-top: var(--pf-c-sequence__control--active--BorderTopWidth) solid var(--pf-c-sequence__control--active--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--active--BorderBottomWidth) solid var(--pf-c-sequence__control--active--BorderBottomColor);
+  }
+
+  &.pf-m-focus,
+  &:focus {
+    border-top: var(--pf-c-sequence__control--focus--BorderTopWidth) solid var(--pf-c-sequence__control--focus--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--focus--BorderBottomWidth) solid var(--pf-c-sequence__control--focus--BorderBottomColor);
+  }
+
+  @media screen and (min-width: $pf-global--breakpoint--xl) {
+    display: none;
+  }
+}
+
+.pf-c-sequence__control-m-start {
+  left: var(--pf-c-sequence__control-m-start--PositionLeft);
+  order: 1;
+  border-right: var(--pf-c-sequence__control-m-start--BorderRightWidth) solid var(--pf-c-sequence__control-m-start--BorderRightColor);
+  border-left: 0;
+}
+
+.pf-c-sequence__control-m-end {
+  right: var(--pf-c-sequence__control-m-end--PositionRight);
+  order: 3;
+  border-right: 0;
+  border-left: var(--pf-c-sequence__control-m-end--BorderLeftWidth) solid var(--pf-c-sequence__control-m-end--BorderLeftColor);
+}
+
+.pf-c-sequence__control-icon {
+  min-width: var(--pf-c-sequence__control-icon--MinWidth);
+  color: var(--pf-c-sequence__control-icon--Color);
```

Comment:
I think you get this for free so it's not needed here.

Reply:
yep!